lib/inc/hocon/config.hpp

```diff
@@ -570,6 +570,9 @@ namespace hocon {
         virtual unwrapped_value get_any_ref(std::string const& path) const;
         virtual std::shared_ptr<const config_value> get_value(std::string const& path) const;
 
+        template<typename T>
+        std::vector<T> get_homogeneous_unwrapped_list(std::string const& path) const;
```

Comment:
I was actually planning to use it as part of the access layer I'm writing for the Facter stuff, to allow just writing one template function there instead of calling into all those explicitly named functions that HOCON is supposed to provide. But I do realize I've done something wrong, because I'm fighting the linker again, as usual xD